src/main/resources/scripts/advanced god mode.sk

```diff
@@ -0,0 +1,36 @@
+#
+# Toggleable god mode
+#
+
+options:
+    permission: command.god # the permission required to use /god
+    disable on join: true # if god mode should be disabled when the user joins
+    disallow pvp: true # if users in god mode shouldn't be allowed to pvp
+
+    enabled: &7Enabled god mode. # when the user enables god mode
+    disabled: &7Disabled god mode. # when the user disables god mode
+    currently enabled: &7You have god mode enabled so you can't hit other users. # when the user tries to pvp in god mode while <disallow pvp> is true
+
+on join:
+    if {@disable on join} is true:
+        delete {god::%player%}
+        send "{@disabled}" to player
+
+on damage of player:
+    if {@disallow pvp} is true:
+        {god::%attacker%} is set
+        send "{@currently enabled}" to attacker
+        cancel event
+    if {god::%victim%} is set:
+    	cancel event
+
+command /god <offline player=%player%>:
+    description: Toggle god mode
+    permission: {@permission}
+    trigger:
+        if {god::%arg%} is not set:
+            send "{@enabled}" to arg
+            set {god::%arg%} to true
+        else if {god::%arg%} is set:
```

Comment:
You could just use `else` here

Reply:
that wouldn't work

Reply:
If something is not set then `else` means it _is_ set...

Reply:
> If something is not set then `else` means it _is_ set...

yes but the variable is set before it sees the else, so it runs what happens in the else as well. else if solves this issue.

Reply:
no it won't, if the first condition is met, it won't continue to the else, the else only catches if the first condition is not met

Reply:
> no it won't, if the first condition is met, it won't continue to the else, the else only catches if the first condition is not met

nope

Reply:
just close the PR if you still won't accept the truth

Reply:
> > no it won't, if the first condition is met, it won't continue to the else, the else only catches if the first condition is not met
> 
> nope

Then its obvious you don't understand how if/else if/else works. 
As blue stated you should close this PR since you dont understand this conditional concept.

Reply:
The most funniest thing is that even if what he said was true, the variable would be set and thus what he said would still be wrong xDD